docs/conf.py

```diff
@@ -1,6 +1,12 @@
 # -*- coding: utf-8 -*-
 
 import  sys, os
+
+from pathlib import Path
+
+sys.path.append(str(Path(__file__).parent))
+sys.path.append(str(Path(__file__).parent.parent))
```

Comment:
So it is possible to import shpinx_build_tools without this 

`SITE_PYTHON ?=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))site-python`

(I understood why it was required after I moved sphinx_build_tools.py )

Reply:
For me it is a question of architecture.  This conf.py is a part of the documentation.  The rules to build documentation are defined in `./utils` [makefile.sphinx](https://github.com/searx/searx/blob/master/utils/makefile.sphinx) and [site-python/sphinx_build_tools.py](https://github.com/searx/searx/blob/master/utils/site-python/sphinx_build_tools.py) is a part of [makefile.sphinx](https://github.com/searx/searx/blob/master/utils/makefile.sphinx).

The makefile.python file does all the python stuff, one task is to make python implementations from the build tools available in build processes.  This is done by extending the `PYTHONPATH`

https://github.com/searx/searx/blob/006f206dc9ff30893cfb6e431c2ee3a7f7768973/utils/makefile.python#L6-L7